### DISCUSSION TITLE
Pixel Watch 3 - no face is available

### DISCUSSION THREAD

**Discussion by bleez99** [NightscoutFoundation/xDrip#3683] [Bug reports]
I can install xDrip on my new watch, when I open preferences, all appears to be normal. 
There was an initial complaint about compatibility with Android 14 - "This application was built for an older version of Android." Wear installer worked perfectly. I do not see the face in the Watch app or on the watch itself.

**Comment by PhilTU** [comment]
Google pulled something, Samsung still has it. See https://github.com/NightscoutFoundation/xDrip/discussions/3581

**Reply by bleez99** [reply]
Yes, they pulled Wear OS 5.  Bummer.  No support for that here yet.

**Reply by bleez99** [reply]
I'd settle for a complication - not available either.

**Reply by bleez99** [reply]
Today, with the September 27 build, I see complications available. Unless I completely missed something while playing with the Google Watch app...

I'm happy I can just check my sugar with the watch now.

**Comment by ELY3M** [comment]
following.  I would like to have a tile from xdrip
